### ISSUE TITLE
Option 1 exception handler not working in Larvel 5.3

### ISSUE THREAD

**Issue by computerfr33k** [flugger/laravel-responder#11]
```php
<?php

namespace App\Exceptions;

use Exception;
use Illuminate\Auth\AuthenticationException;
//use Illuminate\Foundation\Exceptions\Handler as ExceptionHandler;
use Flugg\Responder\Exceptions\Handler as ExceptionHandler;

class Handler extends ExceptionHandler
{
    /**
     * A list of the exception types that should not be reported.
     *
     * @var array
     */
    protected $dontReport = [
        \Illuminate\Auth\AuthenticationException::class,
        \Illuminate\Auth\Access\AuthorizationException::class,
        \Symfony\Component\HttpKernel\Exception\HttpException::class,
        \Illuminate\Database\Eloquent\ModelNotFoundException::class,
        \Illuminate\Session\TokenMismatchException::class,
        \Illuminate\Validation\ValidationException::class,
    ];

    /**
     * Report or log an exception.
     *
     * This is a great spot to send exceptions to Sentry, Bugsnag, etc.
     *
     * @param  \Exception  $exception
     * @return void
     */
    public function report(Exception $exception)
    {
        parent::report($exception);
    }

    /**
     * Render an exception into an HTTP response.
     *
     * @param  \Illuminate\Http\Request  $request
     * @param  \Exception  $exception
     * @return \Illuminate\Http\Response
     */
    public function render($request, Exception $exception)
    {
        return parent::render($request, $exception);
    }

    /**
     * Convert an authentication exception into an unauthenticated response.
     *
     * @param  \Illuminate\Http\Request  $request
     * @param  \Illuminate\Auth\AuthenticationException  $exception
     * @return \Illuminate\Http\Response
     */
    protected function unauthenticated($request, AuthenticationException $exception)
    {
        if ($request->expectsJson()) {
            return response()->json(['error' => 'Unauthenticated.'], 401);
        }

        return redirect()->guest('login');
    }
}
```

And if I use the following exception:
```php
<?php

namespace App\Exceptions;

use Flugg\Responder\Exceptions\ApiException;

class ModelNotFoundException extends ApiException
{
    protected $statusCode = 404;

    protected $errorCode = 'model_not_found';
}
```
```php
Route::get('/', function () {

    throw new \App\Exceptions\ModelNotFoundException('Model not found!');
});
```

I will just get an html page with `Call to undefined method App\Exceptions\Handler::renderApiErrors()`

Though I'm not sure if this package is officially supported on Laravel 5.3.

**Comment by flugger** [comment]
Ah, yeah, haven't had the time to look into Laravel 5.3 support yet. I know the exception handler has changed a bit from 5.2. It should, however, be quite an easy upgrade. I will have a look at it this week and keep you posted! 

I believe option 2 should still be working in the mean time.

**Comment by flugger** [comment]
This should be fixed in v1.1.3, a `composer update` should do it.

Thanks for notifying me about the issue!
